src/main/java/evolvio/Application.java

```diff
@@ -0,0 +1,14 @@
+package evolvio;
+
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.ConfigurableApplicationContext;
+
+@SpringBootApplication
+public class Application {
+    public static void main(String[] args) {
+        SpringApplicationBuilder builder = new SpringApplicationBuilder(Application.class);
+        builder.headless(false);
+        ConfigurableApplicationContext context = builder.run(args);
```

Comment:
public static void main(String[] args) {
        new SpringApplicationBuilder(Application.class).headless(false).run(args);
    }

Reply:
Not sure what this adds